### ISSUE TITLE
PhantomJS crashes (version 1.9.2 on Linux x86_64)

### ISSUE THREAD

**Issue by smedinaw4l** [ariya/phantomjs#11855]
While using CasperJS I managed to make PhantomJS crash every time.
I guess I could provide the script.

Here is the stack trace, generated as indicated in https://github.com/ariya/phantomjs/wiki/Crash-Reporting.

```
Operating system: Linux
                  0.0.0 Linux 2.6.35.4-rscloud #8 SMP Mon Sep 20 15:54:33 UTC 2010 x86_64
CPU: amd64
     family 16 model 4 stepping 2
     4 CPUs

Crash reason:  SIGSEGV
Crash address: 0x38

Thread 0 (crashed)
 0  phantomjs!QWebFrame::evaluateJavaScript [qwebframe.cpp : 1551 + 0x4]
    rbx = 0x00007fffd9cff0c0   r12 = 0x0000000004ad6b10
    r13 = 0x00007fffd9cff210   r14 = 0x00007fffd9cff0b0
    r15 = 0x0000000004876e00   rip = 0x000000000047f960
    rsp = 0x00007fffd9cfef90   rbp = 0x00007fffd9cff140
    Found by: given as instruction pointer in context
 1  phantomjs!WebPage::evaluateJavaScript [webpage.cpp : 683 + 0x13]
    rbx = 0x0000000004ad6b00   r12 = 0x0000000004ad6b10
    r13 = 0x00007fffd9cff210   r14 = 0x00007fffd9cff0c0
    r15 = 0x0000000004876e00   rip = 0x000000000041f1b9
    rsp = 0x00007fffd9cff0b0   rbp = 0x00007fffd9cff140
    Found by: call frame info
 2  phantomjs!WebPage::qt_static_metacall [moc_webpage.cpp : 224 + 0xe]
    rbx = 0x0000000000000000   r12 = 0x0000000004876e00
    r13 = 0x0000000000000011   r14 = 0x0000000000000001
    r15 = 0x0000000001d5fd00   rip = 0x000000000046aa42
    rsp = 0x00007fffd9cff210   rbp = 0x00007fffd9cffbd0
    Found by: call frame info
 3  phantomjs!WebPage::qt_metacall [moc_webpage.cpp : 338 + 0xe]
    rbx = 0x0000000000000000   r12 = 0x0000000004876e00
    r13 = 0x0000000000000011   r14 = 0x0000000000000001
    r15 = 0x0000000001d5fd00   rip = 0x000000000046aebf
    rsp = 0x00007fffd9cff340   rbp = 0x00007fffd9cffbd0
    Found by: call frame info
 4  phantomjs!JSC::Bindings::QtRuntimeMetaMethod::call [qt_runtime.cpp : 1455 + 0x17]
    rbx = 0x00007fffd9cff9a0   r12 = 0x00007fffd9cffbd0
    r13 = 0x00007f1d609d63d0   r14 = 0x0000000000000001
    r15 = 0x0000000001d5fd00   rip = 0x0000000000552f07
    rsp = 0x00007fffd9cff4d0   rbp = 0x0000000000000002
    Found by: call frame info
 5  phantomjs!JSC::cti_op_call_NotJSFunction [JITStubs.cpp : 2191 + 0x3]
    rbx = 0x0000000000000000   r12 = 0x00007f1d629dd2f0
    r13 = 0x00007f1d6265c150   r14 = 0x00007fffd9d002d8
    r15 = 0x00007f1d629dd270   rip = 0x0000000000f5d075
    rsp = 0x00007fffd9d00280   rbp = 0x00007fffd9d002e0
    Found by: call frame info
 6  0x7f1d1c02d448
    rbx = 0x00007f1d1c02d383   r12 = 0x00000000000001fe
    r13 = 0x00007f1d629dd270   r14 = 0xffff000000000000
    r15 = 0xffff000000000002   rip = 0x00007f1d1c02d449
    rsp = 0x00007fffd9d002e0   rbp = 0x00007fffd9d00380
    Found by: call frame info
 7  phantomjs!JSC::Interpreter::executeCall [JITCode.h : 77 + 0x16]
    rip = 0x0000000000f2e376   rsp = 0x00007fffd9d00390
    Found by: stack scanning
 8  phantomjs!JSC::call [CallData.cpp : 38 + 0x21]
    rbx = 0x0000000002acad10   r12 = 0x00007f1d62595098
    r13 = 0x00007f1d62595090   r14 = 0x00007f1d663f8150
    r15 = 0x00007fffd9d004b0   rip = 0x0000000000f7291d
    rsp = 0x00007fffd9d00460   rbp = 0x0000000000000000
    Found by: call frame info
 9  phantomjs!WebCore::ScheduledAction::executeFunctionInContext [JSMainThreadExecState.h : 48 + 0xb]
    rbx = 0x0000000002acad10   r12 = 0x00007f1d62595098
    r13 = 0x00007f1d62595090   r14 = 0x00007f1d663f8150
    r15 = 0x00007fffd9d004b0   rip = 0x00000000010130f9
    rsp = 0x00007fffd9d00470   rbp = 0x0000000000000000
    Found by: call frame info
10  phantomjs!WebCore::ScheduledAction::execute [ScheduledAction.cpp : 132 + 0x29]
    rbx = 0x00007f1d6643b400   r12 = 0x00007f1d62595090
    r13 = 0x00007f1d6645e800   r14 = 0x0000000000000001
    r15 = 0x00007f1d6645e8d0   rip = 0x00000000010137cd
    rsp = 0x00007fffd9d005e0   rbp = 0x00007f1d663f8150
    Found by: call frame info
11  phantomjs!WebCore::DOMTimer::fired [DOMTimer.cpp : 135 + 0xb]
    rbx = 0x00007f1d61e83270   r12 = 0x0000000000000000
    r13 = 0x0000000002ac7b08   r14 = 0x0000000000000001
    r15 = 0x00007fffd9d00680   rip = 0x00000000008b4823
    rsp = 0x00007fffd9d00640   rbp = 0x00007f1d6645e8d0
    Found by: call frame info
12  phantomjs!WebCore::ThreadTimers::sharedTimerFiredInternal [ThreadTimers.cpp : 112 + 0x8]
    rbx = 0x00007f1d61e83270   r12 = 0x0000000000000000
    r13 = 0x00000000045cbab0   r14 = 0x00000000000e709e
    r15 = 0x00007fffd9d00e10   rip = 0x0000000000993983
    rsp = 0x00007fffd9d006d0   rbp = 0x00007f1d6645a7a8
    Found by: call frame info
13  phantomjs!QObject::event [qobject.cpp : 1156 + 0x8]
    rbx = 0x00000000048154e0   r12 = 0x00007fffd9d00e10
    r13 = 0x00000000045cbab0   r14 = 0x00000000000e709e
    r15 = 0x00007fffd9d00e10   rip = 0x0000000001cefb94
    rsp = 0x00007fffd9d00710   rbp = 0x00000000048154e0
    Found by: call frame info
14  phantomjs!QApplicationPrivate::notify_helper [qapplication.cpp : 4562 + 0xc]
    rbx = 0x00007fffd9d01090   r12 = 0x00007fffd9d00e10
    r13 = 0x00000000045cbab0   r14 = 0x00000000000e709e
    r15 = 0x00007fffd9d00e10   rip = 0x00000000015decaf
    rsp = 0x00007fffd9d008b0   rbp = 0x00000000048154e0
    Found by: call frame info
15  phantomjs!QApplication::notify [qapplication.cpp : 4423 + 0x17]
    rbx = 0x00007fffd9d01090   r12 = 0x00007fffd9d00e10
    r13 = 0x00007fffd9d01090   r14 = 0x00000000000e709e
    r15 = 0x00007fffd9d00e10   rip = 0x00000000015e793b
    rsp = 0x00007fffd9d008e0   rbp = 0x00000000048154e0
    Found by: call frame info
16  phantomjs!QCoreApplication::notifyInternal [qcoreapplication.cpp : 946 + 0x2]
    rbx = 0x00000000045c91c0   r12 = 0x00007fffd9d00e10
    r13 = 0x00007fffd9d01090   r14 = 0x00000000000e709e
    r15 = 0x00007fffd9d00e10   rip = 0x0000000001cde3a4
    rsp = 0x00007fffd9d00db0   rbp = 0x00000000048154e0
    Found by: call frame info
17  phantomjs!QTimerInfoList::activateTimers [qcoreapplication.h : 231 + 0x14]
    rbx = 0x00000000045cd898   r12 = 0x0000000000000000
    r13 = 0x0000000000000001   r14 = 0x00000000000e709e
    r15 = 0x00007fffd9d00e10   rip = 0x0000000001d07e9d
    rsp = 0x00007fffd9d00e00   rbp = 0x0000000000a1948f
    Found by: call frame info
18  phantomjs!QEventDispatcherUNIX::processEvents [qeventdispatcher_unix.cpp : 941 + 0x7]
    rbx = 0x0000000000000000   r12 = 0x00007fffd9d00ee0
    r13 = 0x00000000045c8560   r14 = 0x0000000000000001
    r15 = 0x00000000045c92d8   rip = 0x0000000001d080b0
    rsp = 0x00007fffd9d00e90   rbp = 0x00000000045cd030
    Found by: call frame info
19  phantomjs!QEventDispatcherQPA::processEvents [qeventdispatcher_qpa.cpp : 246 + 0x4]
    rbx = 0x0000000004883380   r12 = 0x0000000000000000
    r13 = 0x00000000045c8560   r14 = 0x00007fffd9d00f20
    r15 = 0x00000000045c92d8   rip = 0x000000000163a6a4
    rsp = 0x00007fffd9d00ee0   rbp = 0x00000000045cd030
    Found by: call frame info
20  phantomjs!QEventLoop::processEvents [qeventloop.cpp : 149 + 0x1]
    rbx = 0x0000000004883380   r12 = 0x00007fffd9d00fb0
    r13 = 0x00007fffd9d00fc0   r14 = 0x00000000045c9230
    r15 = 0x00000000045c92d8   rip = 0x0000000001cdd485
    rsp = 0x00007fffd9d00f20   rbp = 0x00007fffd9d00f60
    Found by: call frame info
21  phantomjs!QEventLoop::exec [qeventloop.cpp : 200 + 0x4]
    rbx = 0x0000000004883380   r12 = 0x00007fffd9d00fb0
    r13 = 0x00007fffd9d00fc0   r14 = 0x00000000045c9230
    r15 = 0x00000000045c92d8   rip = 0x0000000001cdd8e8
    rsp = 0x00007fffd9d00f50   rbp = 0x00007fffd9d01090
    Found by: call frame info
22  phantomjs!QCoreApplication::exec [qcoreapplication.cpp : 1218 + 0x4]
    rbx = 0x00000000045c91c0   r12 = 0x00007fffd9d01090
    r13 = 0x00007fffd9d01278   r14 = 0x00007fffd9d01000
    r15 = 0x0000000000000000   rip = 0x0000000001cdf79b
    rsp = 0x00007fffd9d00fb0   rbp = 0x00000000045c91c0
    Found by: call frame info
23  phantomjs!main [main.cpp : 121 + 0x4]
    rbx = 0x0000000004613760   r12 = 0x00007fffd9d01090
    r13 = 0x00007fffd9d01278   r14 = 0x00007fffd9d01000
    r15 = 0x0000000000000000   rip = 0x0000000000427cb2
    rsp = 0x00007fffd9d00ff0   rbp = 0x00007fffd9d01228
    Found by: call frame info
24  libc-2.15.so + 0x2252c
    rbx = 0x0000000000000000   r12 = 0x0000000000407cc0
    r13 = 0x00007fffd9d01220   r14 = 0x0000000000000000
    r15 = 0x0000000000000000   rip = 0x00007f1d64c9552d
    rsp = 0x00007fffd9d01150   rbp = 0x0000000000000000
    Found by: call frame info
25  phantomjs + 0x27a5f
    rip = 0x0000000000427a60   rsp = 0x00007fffd9d01170
    Found by: stack scanning

Thread 1
 0  libc-2.15.so + 0xb961d
    rbx = 0x00007f1d63dddd60   r12 = 0x00007f1d63dddce0
    r13 = 0x00007f1d66537a20   r14 = 0x0000000002aaa1a0
    r15 = 0x0000000002aaa170   rip = 0x00007f1d64d2c61d
    rsp = 0x00007f1d63dddc30   rbp = 0x00000000ffffffff
    Found by: given as instruction pointer in context
 1  libc-2.15.so + 0xb94ae
    rip = 0x00007f1d64d2c4af   rsp = 0x00007f1d63dddc40
    Found by: stack scanning
 2  phantomjs + 0x18b5c3f
    rip = 0x0000000001cb5c40   rsp = 0x00007f1d63dddc48
    Found by: stack scanning
 3  ld-2.15.so + 0x8f89
    rip = 0x00007f1d66337f8a   rsp = 0x00007f1d63dddc70
    Found by: stack scanning
 4  ld-2.15.so + 0x9693
    rip = 0x00007f1d66338694   rsp = 0x00007f1d63dddcb0
    Found by: stack scanning
 5  libpthread-2.15.so + 0x1045f
    rip = 0x00007f1d6583f460   rsp = 0x00007f1d63dddcd8
    Found by: stack scanning
 6  libc-2.15.so + 0x11787
    rip = 0x00007f1d64c84788   rsp = 0x00007f1d63dddd28
    Found by: stack scanning
 7  phantomjs!WTF::TCMalloc_PageHeap::scavengerThread [FastMalloc.cpp : 2499 + 0x4]
    rip = 0x00000000004dda2e   rsp = 0x00007f1d63ddde20
    Found by: stack scanning
 8  phantomjs!WTF::TCMalloc_PageHeap::runScavengerThread [FastMalloc.cpp : 1618 + 0x4]
    rbx = 0x0000000000000000   r12 = 0x00007f1d65841620
    r13 = 0x00007f1d63dde9c0   r14 = 0x00007f1d66551000
    r15 = 0x0000000000000003   rip = 0x00000000004ddc99
    rsp = 0x00007f1d63dddea0   rbp = 0x0000000000000000
    Found by: call frame info
 9  libpthread-2.15.so + 0x7da5
    rbx = 0x0000000000000000   r12 = 0x00007f1d65841620
    r13 = 0x00007f1d63dde9c0   r14 = 0x00007f1d66551000
    r15 = 0x0000000000000003   rip = 0x00007f1d65836da6
    rsp = 0x00007f1d63dddeb0   rbp = 0x0000000000000000
    Found by: call frame info
10  libpthread-2.15.so + 0x1261f
    rip = 0x00007f1d65841620   rsp = 0x00007f1d63dddf58
    Found by: stack scanning
11  libc-2.15.so + 0xebb2c
    rip = 0x00007f1d64d5eb2d   rsp = 0x00007f1d63dddfc0
    Found by: stack scanning

Thread 2
 0  libc-2.15.so + 0xe5253
    rbx = 0x0000000000000000   r12 = 0x00007f1d635dcd80
    r13 = 0x000000000000000b   r14 = 0x00007f1d5c000a98
    r15 = 0x00007f1d5c000d30   rip = 0x00007f1d64d58253
    rsp = 0x00007f1d635dcb50   rbp = 0x00007f1d5c0008e0
    Found by: given as instruction pointer in context
 1  phantomjs!qt_safe_select [qcore_unix.cpp : 83 + 0x15]
    rip = 0x0000000001d062f7   rsp = 0x00007f1d635dcb60
    Found by: stack scanning
 2  phantomjs!QEventDispatcherUNIXPrivate::doSelect [qeventdispatcher_unix.cpp : 223 + 0xd]
    rbx = 0x00007f1d5c0012b0   r12 = 0x00007f1d635dcd80
    r13 = 0x00007f1d5c0008e0   r14 = 0x0000000000000000
    r15 = 0x00007f1d5c000d30   rip = 0x0000000001d075b7
    rsp = 0x00007f1d635dcbf0   rbp = 0x00007f1d5c0008e0
    Found by: call frame info
 3  phantomjs!QEventDispatcherUNIX::processEvents [qeventdispatcher_unix.cpp : 937 + 0x4]
    rbx = 0x00007f1d5c0012b0   r12 = 0x00007f1d635dcd80
    r13 = 0x00007f1d5c0008c0   r14 = 0x0000000000000001
    r15 = 0x00000000046176d8   rip = 0x0000000001d08062
    rsp = 0x00007f1d635dcd30   rbp = 0x00007f1d5c0008e0
    Found by: call frame info
 4  phantomjs!QEventLoop::processEvents [qeventloop.cpp : 149 + 0x1]
    rbx = 0x00007f1d5c0012b0   r12 = 0x00007f1d635dce10
    r13 = 0x00007f1d635dce20   r14 = 0x00000000045c9230
    r15 = 0x00000000046176d8   rip = 0x0000000001cdd485
    rsp = 0x00007f1d635dcd80   rbp = 0x00007f1d635dcdc0
    Found by: call frame info
 5  phantomjs!QEventLoop::exec [qeventloop.cpp : 200 + 0x4]
    rbx = 0x00007f1d5c0012b0   r12 = 0x00007f1d635dce10
    r13 = 0x00007f1d635dce20   r14 = 0x00000000045c9230
    r15 = 0x00000000046176d8   rip = 0x0000000001cdd8e8
    rsp = 0x00007f1d635dcdb0   rbp = 0x00007fffd9d01090
    Found by: call frame info
 6  phantomjs!QThread::exec [qthread.cpp : 542 + 0xc]
    rbx = 0x0000000004617650   r12 = 0x00000000046176d8
    r13 = 0x0000000000000001   r14 = 0x00007f1d66551000
    r15 = 0x0000000000000003   rip = 0x0000000001bdfdd6
    rsp = 0x00007f1d635dce10   rbp = 0x00000000046176d8
    Found by: call frame info
 7  phantomjs!QThreadPrivate::start [qthread_unix.cpp : 338 + 0x9]
    rbx = 0x00000000046176d8   r12 = 0x0000000004614580
    r13 = 0x0000000000000001   r14 = 0x00007f1d66551000
    r15 = 0x0000000000000003   rip = 0x0000000001be2ca2
    rsp = 0x00007f1d635dce60   rbp = 0x0000000004620a00
    Found by: call frame info
 8  libpthread-2.15.so + 0x7da5
    rbx = 0x0000000000000000   r12 = 0x00007fffd9d00570
    r13 = 0x00007f1d635dd9c0   r14 = 0x00007f1d66551000
    r15 = 0x0000000000000003   rip = 0x00007f1d65836da6
    rsp = 0x00007f1d635dceb0   rbp = 0x0000000000000000
    Found by: call frame info

Thread 3
 0  libc-2.15.so + 0xe5253
    rbx = 0x00007f1d61cd1d30   r12 = 0x0000000000a1948f
    r13 = 0x0000000000000016   r14 = 0x00007f1d14000a98
    r15 = 0x00007f1d14000d30   rip = 0x00007f1d64d58253
    rsp = 0x00007f1d61cd1b50   rbp = 0x00000000000d8394
    Found by: given as instruction pointer in context
 1  phantomjs!qt_safe_select [qcore_unix.cpp : 93 + 0x16]
    rip = 0x0000000001d061f8   rsp = 0x00007f1d61cd1b60
    Found by: stack scanning
 2  phantomjs!QEventDispatcherUNIXPrivate::doSelect [qeventdispatcher_unix.cpp : 223 + 0xd]
    rbx = 0x00007f1d140012b0   r12 = 0x00007f1d61cd1d80
    r13 = 0x00007f1d140008e0   r14 = 0x00007f1d61cd1d30
    r15 = 0x00007f1d14000d30   rip = 0x0000000001d075b7
    rsp = 0x00007f1d61cd1bf0   rbp = 0x00007f1d140008e0
    Found by: call frame info
 3  phantomjs!QEventDispatcherUNIX::processEvents [qeventdispatcher_unix.cpp : 937 + 0x4]
    rbx = 0x00007f1d140012b0   r12 = 0x00007f1d61cd1d80
    r13 = 0x00007f1d140008c0   r14 = 0x0000000000000001
    r15 = 0x0000000004826a98   rip = 0x0000000001d08062
    rsp = 0x00007f1d61cd1d30   rbp = 0x00007f1d140008e0
    Found by: call frame info
 4  phantomjs!QEventLoop::processEvents [qeventloop.cpp : 149 + 0x1]
    rbx = 0x00007f1d140012b0   r12 = 0x00007f1d61cd1e10
    r13 = 0x00007f1d61cd1e20   r14 = 0x00000000045c9230
    r15 = 0x0000000004826a98   rip = 0x0000000001cdd485
    rsp = 0x00007f1d61cd1d80   rbp = 0x00007f1d61cd1dc0
    Found by: call frame info
 5  phantomjs!QEventLoop::exec [qeventloop.cpp : 200 + 0x4]
    rbx = 0x00007f1d140012b0   r12 = 0x00007f1d61cd1e10
    r13 = 0x00007f1d61cd1e20   r14 = 0x00000000045c9230
    r15 = 0x0000000004826a98   rip = 0x0000000001cdd8e8
    rsp = 0x00007f1d61cd1db0   rbp = 0x00007fffd9d01090
    Found by: call frame info
 6  phantomjs!QThread::exec [qthread.cpp : 542 + 0xc]
    rbx = 0x0000000004826a10   r12 = 0x0000000004826a98
    r13 = 0x0000000000000001   r14 = 0x00007f1d66551000
    r15 = 0x0000000000000003   rip = 0x0000000001bdfdd6
    rsp = 0x00007f1d61cd1e10   rbp = 0x0000000004826a98
    Found by: call frame info
 7  phantomjs!QThreadPrivate::start [qthread_unix.cpp : 338 + 0x9]
    rbx = 0x0000000004826a98   r12 = 0x00000000048b8530
    r13 = 0x0000000000000001   r14 = 0x00007f1d66551000
    r15 = 0x0000000000000003   rip = 0x0000000001be2ca2
    rsp = 0x00007f1d61cd1e60   rbp = 0x00000000048312b0
    Found by: call frame info
 8  libpthread-2.15.so + 0x7da5
    rbx = 0x0000000000000000   r12 = 0x00007fffd9cfc860
    r13 = 0x00007f1d61cd29c0   r14 = 0x00007f1d66551000
    r15 = 0x0000000000000003   rip = 0x00007f1d65836da6
    rsp = 0x00007f1d61cd1eb0   rbp = 0x0000000000000000
    Found by: call frame info

Loaded modules:
0x00400000 - 0x026b9fff  phantomjs  ???  (main)
0x7f1d60270000 - 0x7f1d602c8fff  DejaVuSerif.ttf  ???
0x7f1d602c9000 - 0x7f1d6031afff  DejaVuSansMono.ttf  ???
0x7f1d60a03000 - 0x7f1d60a9dfff  DejaVuSansCondensed-Bold.ttf  ???
0x7f1d60a9e000 - 0x7f1d60b3bfff  DejaVuSansCondensed.ttf  ???
0x7f1d60e9b000 - 0x7f1d610a0fff  libnss_dns-2.15.so  ???
0x7f1d610a1000 - 0x7f1d612adfff  libnss_files-2.15.so  ???
0x7f1d612ae000 - 0x7f1d614c2fff  libresolv-2.15.so  ???
0x7f1d63ddf000 - 0x7f1d64047fff  libssl.so.1.0.0  ???
0x7f1d64049000 - 0x7f1d6441afff  libcrypto.so.1.0.0  ???
0x7f1d6441f000 - 0x7f1d64621fff  locale-archive  ???
0x7f1d64622000 - 0x7f1d6484cfff  libexpat.so.1.6.0  ???
0x7f1d6484d000 - 0x7f1d64a5cfff  libbz2.so.1.0.6  ???
0x7f1d64a5d000 - 0x7f1d64c72fff  libz.so.1.2.7  ???
0x7f1d64c73000 - 0x7f1d65019fff  libc-2.15.so  ???
0x7f1d6501e000 - 0x7f1d65233fff  libgcc_s.so.1  ???
0x7f1d65234000 - 0x7f1d6552afff  libm-2.15.so  ???
0x7f1d6552b000 - 0x7f1d65819fff  libstdc++.so.6.0.16  ???
0x7f1d6582f000 - 0x7f1d65a47fff  libpthread-2.15.so  ???
0x7f1d65a4c000 - 0x7f1d65c54fff  librt-2.15.so  ???
0x7f1d65c55000 - 0x7f1d65e58fff  libdl-2.15.so  ???
0x7f1d65e59000 - 0x7f1d6608efff  libfontconfig.so.1.4.4  ???
0x7f1d6608f000 - 0x7f1d6632efff  libfreetype.so.6.8.1  ???
0x7f1d6632f000 - 0x7f1d66350fff  ld-2.15.so  ???
0x7f1d66546000 - 0x7f1d6654efff  12b26b760a24f8b4feb03ad48a333a72-le64.cache-3  ???
0x7fffd9dff000 - 0x7fffd9dfffff  linux-gate.so  ???
```

**Comment by azolotko** [comment]
Should already be fixed in 2.0.0 with update to Qt 5. At least it no longer has the exact same line that caused the crash.

**Comment by Vitallium** [comment]
Thanks! Closing this!
